lib/language.js

```diff
@@ -4389,29 +4389,37 @@ function init() {
       ,it: 'Lingua'
       }
     ,'Add new' : {
-      cs: 'Přidat nový'
+      cs: 'Přidat nový',
+      ,ro: 'Adaugă nou'
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expected an assignment or function call and instead saw an expression.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1287885599/issues/source?bid=2452379&fileBranchId=2945886#l4393)